fastos/src/tests/tests.h

```diff
@@ -43,7 +43,7 @@ class BaseTest : public FastOS_Application
    bool Progress (bool result, const char *str)
    {
       char string[MAX_STR_LEN];
-      sprintf(string, "%s: %s\n",
+      snprintf(string, sizeof(string) - 1, "%s: %s\n",
```

Comment:
Why -1 ?. The size passed to snprintf includes the NUL byte.

Reply:
That was to avoid warning. But as you point out the ensuring of zero termination was missing.
Added.